### PR TITLE
Added UploadConversionWithIdentifiers example

### PR DESCRIPTION
**Pull request by fiboknacky**
Also updated the UploadConversionEnhancement example.

### CODE REVIEW

File: examples/Remarketing/UploadConversionWithIdentifiers.php [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+    private const CONVERSION_ACTION_ID = 'INSERT_CONVERSION_ACTION_ID_HERE';
+    private const EMAIL_ADDRESS = 'INSERT_EMAIL_ADDRESS_HERE';
+    // The date time at which the conversion occurred.
+    // Must be after the click time, and must include the time zone offset.
+    // The format is "yyyy-mm-dd hh:mm:ss+|-hh:mm", e.g. '2019-01-01 12:32:45-08:00'.
+    private const CONVERSION_DATE_TIME = 'INSERT_CONVERSION_DATE_TIME_HERE';
+    private const CONVERSION_VALUE = 'INSERT_CONVERSION_VALUE_HERE';
+
+    // Optional: Specifies the order ID.
+    private const ORDER_ID = 'INSERT_ORDER_ID_HERE';
```
Comment (wfansh) [inline examples/Remarketing/UploadConversionWithIdentifiers.php:54]: should ORDER_ID be null instead?
it's optional

File: examples/Remarketing/UploadConversionEnhancement.php [hunk 2]
```diff
@@ -241,7 +241,7 @@ public static function runExample(
         // Prints the status message if any partial failure error is returned.
         // Note: The details of each partial failure error are not printed here, you can refer to
         // the example HandlePartialFailure.php to learn more.
-        if (!is_null($response->getPartialFailureError())) {
```
Comment (PierrickVoulet) [inline examples/Remarketing/UploadConversionEnhancement.php:244]: I could find 5 more occurrences of this in other code examples, could you change them as well in this PR?

File: examples/Remarketing/UploadConversionWithIdentifiers.php (hunk 1 above)
Comment (PierrickVoulet) [reply examples/Remarketing/UploadConversionWithIdentifiers.php:54]: +1

File: examples/Remarketing/UploadConversionWithIdentifiers.php [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+            $clickConversion->setOrderId($orderId);
+        }
+
+        // Uses the SHA-256 hash algorithm for hashing user identifiers in a privacy-safe way, as
+        // described at https://support.google.com/google-ads/answer/9888656.
+        $hashAlgorithm = "sha256";
+
+        // Creates a user identifier to store the hashed email address.
+        $userIdentifier = new UserIdentifier([
+            // Creates a user identifier using the hashed email address.
```
Comment (PierrickVoulet) [inline examples/Remarketing/UploadConversionWithIdentifiers.php:160]: This should be removed (duplicate of line 158)

**Review by PierrickVoulet — CHANGES_REQUESTED** [review]

**Review by wfansh — APPROVED** [review]

**Review by PierrickVoulet — APPROVED** [review]

**Pull request merged**